### PR TITLE
Refactor documentation to be more discoverable and easier to get started

### LINKED ISSUE #852: Refactor documentation across solution projects under a unified location
Right now we have a discoverability issue with our documentation, even if we are providing guidance or a solution to a problem it is difficult for users to find. 

- [x] Move all docs under `\root\docs\`
    * As it stands docs are stored under both `root\solutions\virtual-assistant\docs` and `\root\solutions\analytics` but as we have seen these already have direct overlap.
     * This provides the added benefit of integrating with [Github Pages](https://help.github.com/en/categories/github-pages-basics) in the future if we wish.
- [x] Reorganize docs directories into related buckets
    * Analytics, Skills, Virtual Assistant, Media, Transcripts
- [x] Standardize skill documentation
- [x] Update skill docs with latest scenario & language model scenarios
- [x] Consolidate Virtual Assistant pages for small docs
    * There are a few docs with a short topic that can be better served
- [x] Add an FAQ page highlighting common issues, broken down into categories
    * 💡 Add in the Virtual Assistant master code as a QnA Maker instance
    * Link to partner repositories for some questions
- [x] Add link to analytics in main readme
- [x] Add unit test overview to the VA testing doc
- [ ] Add best practices for developing skills

### PR DESCRIPTION
**Pull request by ryanisgrig**
## Description

This is a large refactor of where we store our docs to make them easier to get started and discoverable to new users. I'm opening this as a draft PR for now, I appreciate any feedback you have on the new layout/content.

### Maintenance
Close #852 
- [x] Move all docs under `\root\docs\`
- [x] Reorganize docs directories into related buckets
    * Analytics, Skills, Virtual Assistant, Media, Transcripts
- [x] Standardize skill documentation
- [x] Update skill docs with latest scenario & language model scenarios
- [x] Consolidate Virtual Assistant pages for small docs
- [x] Add link to analytics in main readme
- [x] Add unit test overview to the VA testing doc
- [x] Add best practices for developing skills

## Checklist

- [x] I have updated related documentation

### CODE REVIEW

**Review by lzc850612 — APPROVED** [review]

**Review by lauren-mills — APPROVED** [review]

File: docs/virtual-assistant/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+[ToDo Skill](../skills/productivity-todo.md) | The *preview* Skill provides a comprehensive language model for common task related activities, The Skill is currently integrated into Microsoft Graph (outlookTask) and Google APIs.
+[Point of Interest Skill](../skills/pointofinterest.md) | The *preview* Skill provides a comprehensive language model for finding points of interest and requesting directions. The Skill currently provides integration into Azure Maps and Foursquare.
+[Automotive Skill](../skills/automotive.md) | The *preview* Skill provides Vehicle Setting control support for an assistant along with an initial set of common vehicle settings which can be extended further. At this time the Skill surfaces device events to demonstrate how device control could be achieved.
+[Experimental Skills](../skills/experimental-Skills.md) | Experimental Skills are early prototypes of Skills to help bring Skill concepts to life for demonstrations and proof-of-concepts along with providing different examples to get you  started. Restaurant Booking and News Skills are currently available.
```
Comment (darrenj) [inline docs/virtual-assistant/README.md:61]: broken link

**Review by darrenj — CHANGES_REQUESTED** [review]
I've done a pass through, big improvement! Thanks for doing this. We should probably look to add transcripts to all skill documentation (eg productivity too)

File: docs/virtual-assistant/README.md (hunk 1 above)
Comment (darrenj) [reply docs/virtual-assistant/README.md:61]: Also this functionality isn't there yet in calendar :)

Find a Meeting Room 
•	Book a conference room
•	Can you reserve room 258 on Thursday at 1 PM?

**Comment by ryanisgrig** [comment]
@darrenj updated docs addressing your comments & merged in the changes happening on master branch.

**Review by darrenj — APPROVED** [review]

**Comment by darrenj** [comment]
PR issue is due to this branch being out of sync with main code branch and changes to skill welcome message. Should be resolved once merged

**Pull request merged**